### ISSUE TITLE
Remove icinga-studio packages

### ISSUE THREAD

**Issue by dnsmichi** [Icinga/icinga-packaging#92]
icinga/icinga2#6059 removes Icinga Studio in v2.9.0. Packages need to updated once this PR is merged into master.

Affected:

- [x] Fedora
- [x] Ubuntu
- [x] Debian

**Comment by lazyfrosch** [comment]
Removed in:
* https://github.com/Icinga/deb-icinga2/commit/a376d26cb7f2bfdb7a414e32aa96cdcd3e9d8c4e
* https://github.com/Icinga/rpm-icinga2/commit/bd7b8a5ea3086e3c155fbab1b683bd14f6cdd008
